src/app/app-routing.module.ts

```diff
@@ -1,11 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { trimEnd } from 'lodash';
+
 import { LoginComponent } from './login/login.component';
 import { SigninComponent } from './signin/signin.component';
 import { ContextResolver } from './shared/context-resolver.service';
 
 
+export function removeAction(url: string) {
```

Comment:
This would be better as a Directive or a Pipe. I don't think it really goes in the app.routing.module